modules/popup.js

```diff
@@ -0,0 +1,133 @@
+/* eslint-disable no-prototype-builtins */
+import {
+  pokemonApi,
+} from './load.js';
+import pokemons from './pokemons.js';
+
+const commentApi = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Ia58UmcsbOOzLVcpoayr/comments';
+const pokeApi = 'https://pokeapi.co/api/v2/pokemon/';
+
+const popUpMenu = () => {
+  const modal = document.getElementById('modal');
+
+  pokemons.forEach((pokemon) => {
+    const commentBtn = document.getElementById(`comment${pokemon.name}`);
+    commentBtn.addEventListener('click', async () => {
+      const pokeReq = new Request(`${pokeApi}${pokemon.name.toLowerCase()}`);
+      const pokeRes = await fetch(pokeReq);
+      const pokeData = await pokeRes.json();
+
+      const req = new Request(`${commentApi}?item_id=${pokemon.name}`);
```

Comment:
- [ ] It would be better in the video presentation you highlight the piece of code you are proud of as shown in the code review guidelines below

![Screenshot from 2022-07-27 14-55-47](https://user-images.githubusercontent.com/46388113/181280193-a568a1f1-6c9b-4444-ac27-7ad445740146.png)

Reply:
we have adjusted this in the video. Thanks for your time. @addod19